src/app/kpis/page.tsx

```diff
@@ -58,49 +58,119 @@ function LoadingState() {
 }
 
 function KpiDashboard({ kpiData }: { kpiData: KpiData }) {
-  console.log("INSIDE KPI DASH");
-  console.log(kpiData);
+  // console.log("INSIDE KPI DASH");
+  // console.log(kpiData);
   return (
     <>
-      <h1>KPI Dashboard</h1>
-      <p>
-        TOTAL Trade Volume (USD): {displayNumber(kpiData.tradeVolume.total.USD)}
-      </p>
-      <p>TOTAL Trade Volume (XRD): {kpiData.tradeVolume.total.XRD}</p>
-      <LineChart
-        title={"Weekly Trade Volume (XRD)"}
-        x={kpiData.tradeVolume.weekly.XRD.map((o) => o.weekIdentifier)}
-        y={kpiData.tradeVolume.weekly.XRD.map((o) => o.value)}
-      />
-      <LineChart
-        title={"Weekly Trade Volume (USD)"}
-        x={kpiData.tradeVolume.weekly.USD.map((o) => o.weekIdentifier)}
-        y={kpiData.tradeVolume.weekly.USD.map((o) => o.value)}
-      />
-      <h2>Socials</h2>
-      <p>Youtube Subscribers: {kpiData.socials.youtubeSubscribers}</p>
-      <p>Instagram Followers: {kpiData.socials.instagramFollowers}</p>
-      <p>Twitter Followers: {kpiData.socials.twitterFollowers}</p>
-      <LineChart
-        title={"Weekly Website Page Requests"}
-        x={kpiData.website.pageRequests.map((o) => o.weekIdentifier)}
-        y={kpiData.website.pageRequests.map((o) => o.value)}
-      />
-      <LineChart
-        title={"Weekly Website Unique Visitors"}
-        x={kpiData.website.uniqueVisitors.map((o) => o.weekIdentifier)}
-        y={kpiData.website.uniqueVisitors.map((o) => o.value)}
-      />
+      <div className="w-1/2">
+      <h2
+        className="text-md bg-gradient-to-r from-dexter-gradient-blue to-dexter-gradient-green to-50% bg-clip-text text-transparent font-normal"
+        style={{
+          margin: 0,
+          marginBottom: "20px",
+          marginTop: "0px",
+          fontSize: "45px",
+        }}
```

Comment:
Please replace with tailwindcss classes:
``` 
!m-0 !mb-5 !text-[45px] 
```

I know there are issues with default values not being editable but adding `!` (equals `!important`) should fix the issue.